images/win/scripts/Tests/WindowsFeatures.Tests.ps1

```diff
@@ -0,0 +1,25 @@
+Describe "WindowsNETFeatures" {
```

Comment:
Recently, feature `Microsoft-Windows-Subsystem-Linux` was enabled: https://github.com/actions/virtual-environments/pull/1081/files

Could you please add it here?

Reply:
Added